.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
I initially removed it since I was getting this error with travis:
![image](https://cloud.githubusercontent.com/assets/11187343/22615842/177a660a-ea53-11e6-86ec-bc9c577c888e.png)